qlib/workflow/task/manage.py

```diff
@@ -248,9 +252,9 @@ def query(self, query={}, decode=True, task_pool=None):
         for t in task_pool.find(query):
             yield self._decode_task(t)
 
-    def re_query(self, task, task_pool=None):
+    def re_query(self, _id, task_pool=None):
```

Comment:
I think we can remove all the `task_pool` parameter except the one in `__init__`
```
def __init__(...):
     self._task_pool = task_pool

@property
def task_pool(...):
     return mongo instance.
```